src/test/scala/no/ndla/draftapi/repository/UserDataRepositoryTest.scala

```diff
@@ -133,7 +136,11 @@ class UserDataRepositoryTest extends IntegrationSuite with TestEnvironment {
     repository.userDataCount should be(1)
     repository.insert(data2)
     repository.userDataCount should be(2)
-    repository.insert(data3)
+    try {
+      repository.insert(data3)
+    } catch {
+      case _: PSQLException => // Ignore results
+    }
```

Comment:
Denne try-catchen gjør vel ingenting siden funksjonen allerede returnerer en `Try`?

Reply:
Kossen ville du skreve det? Tanken var å sjekke at det blei kasta en exception ved insert på to like user_id.

Reply:
Fiksa det no, meiner eg.